pretty-print.js

```diff
@@ -10,7 +10,7 @@ const { MESSAGE } = require('triple-beam');
  * serializes `info` objects. This was previously exposed as
  * { prettyPrint: true } to transports in `winston < 3.0.0`.
  */
-module.exports = format(function (info, opts) {
-  info[MESSAGE] = inspect(info, false, opts.depth || null, opts.colorize);
+module.exports = format((info, opts = { depth: null }) => {
+  info[MESSAGE] = inspect(info, false, opts.depth, opts.colorize);
```

Comment:
This is technically not equivalent. The default value is not a deep assignment so if `{ colorize: true }` is used the value passed to `inspect` will be `undefined`, not `null` – which is actually significant.